xedocs/schemas/corrections/electron_diffusion_cte.py

```diff
@@ -0,0 +1,19 @@
+'''
+# Correction: Electron diffusion
+# What is it correction: S2 width cut (cutax)
+
+# Description: This correction is to take into account how wide out S2s can be as higher diffusion implies longer S2s. While we do not directly correct our waveforms with this information it is used in cutax for the cut_s2_width cut.
+
+# Latest wiki reference: 
+
+'''
+
+import rframe
+
+from .base_corrections import TimeSampledCorrection
+
+class ElectronDiffusionCte(xd.schemas.base_corrections.TimeSampledCorrection):
```

Comment:
Have you run any tests? this will immediately fail since you use 

`xd.schemas.base_corrections.TimeSampledCorrection` instead of `TimeSampledCorrection`

```suggestion
class ElectronDiffusionCte(TimeSampledCorrection):
```